pkg/statistics/handle/util/ddl_event.go

```diff
@@ -33,18 +38,36 @@ type DDLEvent struct {
 	oldTableInfo *model.TableInfo
 	oldPartInfo  *model.PartitionInfo
 	columnInfos  []*model.ColumnInfo
+	// schemaID is the ID of the schema that the table belongs to.
+	// Used to filter out the system or memory tables.
+	schemaID int64
```

Comment:
If the schema name in DDL jobs is always available, we should use it to simplify our code.